src/NuGet.Core/NuGet.Configuration/Proxy/ProxyCredentialCache.cs

```diff
@@ -1,15 +1,15 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-#if !NETSTANDARD1_5
 using System;
 using System.Collections.Concurrent;
 using System.Net;
 
 namespace NuGet.Configuration
 {
-    public class ProxyCache : IProxyCache
+    public class ProxyCredentialCache : IProxyCredentialCache, ICredentials
```

Comment:
IProxyCache was the same named used in NuGet.Core.dll. Changing this makes it difficult for anyone else using the nuget libraries. I'm not sure how common this is, but I think we should be somewhat cautious about renaming.